src/main/jbake/content/community/index.adoc

```diff
@@ -21,14 +21,15 @@ For more information about available support for TomEE please have a look on lin
 
 === Contribute to This Website
 
-This website is a simple JBake project you can find at https://git-wip-us.apache.org/repos/asf/tomee-site-generator.git.
```

Comment:
Please keep asf repo as the main repository and add github as a subpart or alternative way to work but since it is a proxy we should make explicit that it is synch-ed and therefore not the main source of truth somehow IMHO.